wire-library/wire-tests/src/jvmKotlinInteropTest/proto-kotlin/com/squareup/wire/protos/kotlin/map/Mappy.kt

```diff
@@ -83,8 +85,8 @@ class Mappy(
       Mappy::class, 
       "type.googleapis.com/com.squareup.wire.protos.kotlin.map.Mappy"
     ) {
-      private val thingsAdapter: ProtoAdapter<Map<String, Thing>> =
-          ProtoAdapter.newMapAdapter(ProtoAdapter.STRING, Thing.ADAPTER)
+      private val thingsAdapter: ProtoAdapter<Map<String, Thing>> by lazy(PUBLICATION) {
```

Comment:
I would prefer the same approach be used as Java. This requires a lock, an allocation, and causes creation of `KProperty` objects and a `KProperty[]` array.

Reply:
I've addressed.